src/config/theme.ts

```diff
@@ -55,17 +48,25 @@ declare module '@mui/material/styles' {
       icon?: string;
     };
   }
+  interface GrayscalePaletteColor
+    extends PaletteColor,
+      Pick<Color, 300 | 200 | 100> {
+    tint: string;
+  }
```

Comment:
add for IconButton too? Then you don't need to hard cast to IconButtonProps (in DeleteMutationButton)
```
declare module '@mui/material/IconButton' {
  interface IconButtonPropsColorOverrides {
    grayscale: true;
  }
}
```